tests/integration/test_nitrate.py

```diff
@@ -0,0 +1,85 @@
+import yaml
+import shutil
+import tempfile
+from fmf import Tree
+from pathlib import Path
+from click.testing import CliRunner
+from requre import RequreTestCase
+import tmt.base
+import tmt.cli
+
+# Prepare path to examples
+TEST_DIR = Path(__file__).parent
+NITRATE_EXAMPLE = TEST_DIR.parent.parent / "examples" / "nitrate_cases"
+
+
+# general test plan for this component is: TP#29309
+class NitrateExport(RequreTestCase):
+
+    def setUp(self):
+        super().setUp()
+        self.tmpdir = Path(tempfile.mktemp())
+        shutil.copytree(NITRATE_EXAMPLE, self.tmpdir)
+
+    def tearDown(self):
+        shutil.rmtree(self.tmpdir)
+        super().tearDown()
+
+    def test_create(self):
+        fmf_node = Tree(self.tmpdir).find("/new_testcase")
+        self.assertNotIn("extra-nitrate", fmf_node.data)
+        test = tmt.base.Test(fmf_node)
+        test.export(format_='nitrate', create=True, general=True)
+        # reload the node data to see if it appear there
+        fmf_node = Tree(self.tmpdir).find("/new_testcase")
+        self.assertIn("extra-nitrate", fmf_node.data)
+
+    def test_existing(self):
+        fmf_node = Tree(self.tmpdir).find("/existing_testcase")
+        self.assertEqual(fmf_node.data["extra-nitrate"], "TC#0609686")
+        test = tmt.base.Test(fmf_node)
+        print(test.fmf_id)
+        test.export(format_='nitrate', create=True, general=True)
+
+
+class NitrateImport(RequreTestCase):
+
+    def setUp(self):
+        super().setUp()
+        self.tmpdir = Path(tempfile.mktemp())
+        shutil.copytree(NITRATE_EXAMPLE, self.tmpdir)
+
+    def tearDown(self):
+        shutil.rmtree(self.tmpdir)
+        super().tearDown()
+
+    def test_import_manual_confirmed(self):
+        runner = CliRunner()
+        # TODO: import does not respect --root param anyhow (could)
+        result = runner.invoke(
+            tmt.cli.main, ['--root', self.tmpdir / "import_case", "test", "import", "--nitrate", "--manual", "--case=609704"])
+        self.assertEqual(result.exit_code, 0)
+        self.assertIn("Importing the 'Imported_Test_Case'", result.output)
+        self.assertIn("test case found 'TC#0609704'", result.output)
+        self.assertIn("Metadata successfully stored into", result.output)
+        filename = next(filter(lambda x: "Metadata successfully stored into" in x and "main.fmf" in x,
+                               result.output.splitlines())).split("'")[1]
+        # /home/jscotka/git/tmt/Manual/Imported_Test_Case/main.fmf
+        # TODO: not possible to specify, where to store data, it always creates Manual subdir, I do not want it.
+        self.assertIn("/Manual/Imported_Test_Case/main.fmf", filename)
+        self.assertTrue(Path(filename).exists())
+        with open(Path(filename)) as fd:
+            out = yaml.safe_load(fd)
+            self.assertIn("Tier1", out["tag"])
+            self.assertIn("tmt_test_component", out["component"])
+
+    def test_import_manual_proposed(self):
```

Comment:
What is the point of this test case? The import is done only for `CONFIRMED` tests.

Reply:
it is also mentioned in `TODO`:
I expected that it at least fail in case  this case is not relevant. so that it will not create anything.
So usecase there is that it will prove that it will ignore `PROPOSED` case, with potential issue what it catch (no error for no import)

Reply:
Thanks for clarification. Makes sense that `PROPOSED` should be ignored. What about adding one more import test for regular automated tests? This is (will be) probably the most common use case. So it would be nice to have this covered. But it does not have to be in this pull request.